src/main/navigation/navigation.c

```diff
@@ -2837,7 +2837,13 @@ void updateLandingStatus(timeMs_t currentTimeMs)
 
 bool isLandingDetected(void)
 {
+#if !defined(USE_VARIABLE_PITCH)
     return STATE(AIRPLANE) ? isFixedWingLandingDetected() : isMulticopterLandingDetected();
+#else
+    return STATE(AIRPLANE)
```

Comment:
Probably time to break it into if else statements.

If else can also easily have if(STATE(HELICOPTER)) on a single #if defined() clause, with no else.

Less need to copy and paste code (and risk a bug in the future) as well.

Reply:
You mean:

`#if defined(USE_VARIABLE_PITCH)`
`    if (STATE(HELICOPTER)) {`
`        return isHelicopterLandingDetected();`
`    }`
`#endif`
`    return STATE(AIRPLANE) ? isFixedWingLandingDetected() : isMulticopterLandingDetected();`

Very true! :)

Reply:
Done!